Source/ExcelDnaPack/ExcelDnaPack.csproj

```diff
@@ -32,6 +32,7 @@
     <LangVersion>3</LangVersion>
     <PlatformTarget>x86</PlatformTarget>
     <UseVSHostingProcess>true</UseVSHostingProcess>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
This will change ExcelDnaPack from running as 32-bit to running as 64-bit in most cases. I'm not sure we want to do that, even though it should be fine. Eventually we want it to run as an MSBuild task, which I think normally runs as a 32-bit process, though I'm not sure. At least it needs some thought.

Reply:
In particular, since it calls the Win32 API a bit, I have not checked whether there are 32-bit / 64-bit surprises there.

Reply:
Makes sense to test this out then.

Reply:
Sorry - I only see the `<PlatformTarget>x86</PlatformTarget>` now - I think that means `<Prefer32Bit>false</Prefer32Bit>` is irrelevant.